apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -406,17 +404,10 @@ static Extension createExtensionDate(
       CcwCodebookInterface ccwVariable, Optional<BigDecimal> dateYear) {
 
     Extension extension = null;
-    try {
-      String stringDate = dateYear.get().toString() + "-01-01";
-      Date date1 = new SimpleDateFormat("yyyy-MM-dd").parse(stringDate);
-      DateType dateYearValue = new DateType(date1, TemporalPrecisionEnum.YEAR);
-      String extensionUrl = calculateVariableReferenceUrl(ccwVariable);
-      extension = new Extension(extensionUrl, dateYearValue);
-
-    } catch (ParseException e) {
-      throw new InvalidRifValueException(
-          String.format("Unable to parse reference year: '%s'.", dateYear.get()), e);
-    }
+    String stringDate = dateYear.get().toString();
+    DateType dateYearValue = new DateType(stringDate);
+    String extensionUrl = calculateVariableReferenceUrl(ccwVariable);
+    extension = new Extension(extensionUrl, dateYearValue);
```

Comment:
Youre right. Checking the validity of dateYear is what the try-catch block is trying to do. I should add a different exception to catch since we are not parsing dateYear to create a Date anymore.